test/docker-e2e/e2e_simple_test.go

```diff
@@ -167,7 +167,7 @@ func testPFBSubmission(t *testing.T, chain *tastoradockertypes.Chain, cfg *docke
 
 	// submit blob transaction using TxClient with proper minimum fee
 	// Required: 0.025utia per gas unit, so 200000 * 0.025 = 5000 utia minimum
-	txResp, err := txClient.SubmitPayForBlob(ctx, []*share.Blob{blob}, user.SetGasLimit(200000), user.SetFee(5000))
+	txResp, err := txClient.SubmitPayForBlob(ctx, []*types.Blob{blob}, user.SetGasLimit(200000), user.SetFee(5000))
```

Comment:
if this was required for compilation, why didn't CI fail on PRs before this change?